src/spring/azext_spring/commands.py

```diff
@@ -486,5 +492,26 @@ def load_command_table(self, _):
                             exception_handler=handle_asc_exception) as g:
         g.custom_command('list', 'managed_component_instance_list', validator=validate_instance_list)
 
+    with self.command_group('spring job', custom_command_type=job_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_command('create', 'job_create', is_preview=True, validator=job_validators.validate_job_create)
+        g.custom_command('update', 'job_update', is_preview=True, validator=job_validators.validate_job_update)
+        g.custom_command('delete', 'job_delete', is_preview=True, validator=job_validators.validate_job_delete)
```

Comment:
Do we need to consider adding `confirmation=True` for the delete command?

Reply:
> Do we need to consider adding `confirmation=True` for the delete command?

@zhoxing-ms Not needed. It's aligned with CLI command to delete other resources in Azure Spring Apps for now.